movie/src/components/PopupCard/BoxDisplayGenres/BoxDisplayGenres.jsx

```diff
@@ -0,0 +1,43 @@
+import { Box } from "@mui/material";
+import FilterPopup from "../FilterPopup/FilterPopup";
+import Collapse from "@mui/material/Collapse";
+import "./BoxDisplayGenres.css";
+import { useState } from "react";
+
+const BoxDisplayGenres = ({ genres, expanded, onClick }) => {
+  const [selectedGenres, setSelectedGenres] = useState([]);
+
+  const isClicked = (genre) =>
+    selectedGenres.includes(genre) ? "labelClicked" : "labelUnClick";
+  const handleClickFilter = (genre) => {
+    var genresClicked = [...selectedGenres];
```

Comment:
never never never never never never never 
--------------use `var`-------------------

`const` is a way to go

Reply:
`genresClicked` ---> `genresSelected`

Since it is an Array